native/src/utils/openExternalUrl.ts

```diff
@@ -10,32 +10,33 @@ import sendTrackingSignal from './sendTrackingSignal'
 import { reportError } from './sentry'
 
 const openExternalUrl = async (url: string, showSnackbar: (snackbar: SnackbarType) => void): Promise<void> => {
-  const { protocol } = new URL(url)
+  const encodedUrl = encodeURI(url)
+  const { protocol } = new URL(encodedUrl)
```

Comment:
Personal preference: I always like it a little bit more to name variables that should not be used some way that make that clear, for example something like `unsafeUrl` or `rawUrl` or similar... Otherwise its pretty easy to just type `url` and use that further down, while only `encodedUrl` should be used which would introduce bugs again.